New Item Extension mod element (#2479)
MCreator/MCreator: plugins/mcreator-localization/help/default/item_extension/fuel_success_condition.md

> The condition has to be valid to use the selected item as fuel. If the condition fails, the item will not work as fuel until the condition successes.

Reviewer: Если условие указано, выбранный предмет может являться топливом только при выполнении этого условия.
```suggestion
If condition is specified, the selected item can only be used as fuel if the condition passes.
```
